releasenotes/notes/feat-split-by-line-splitter-aa804cb2346c6ed9.yaml

```
@@ -0,0 +1,3 @@
+enhancements:
```

We need the `---` at the beginning of the file
```suggestion
---
enhancements:
```